datalad_deprecated/resources/website/assets/js/main.js

```diff
@@ -585,6 +678,38 @@ function directory(jQuery, md5) {
   return table;
 }
 
+// https://stackoverflow.com/questions/47207355/copy-to-clipboard-using-jquery/47207504
+function copyToClipboard(text) {
+  var $temp = $('<input>');
+  $('body').append($temp);
+  $temp.val(text).select();
+  document.execCommand('copy');
+  $temp.remove();
+}
+
+function getRenderers(filename) {
+  validRenderers = []
+
+  EXTERNAL_RENDERERS.forEach(function(renderer) {
```

Comment:
I was on the edge on this, but for the clarity lets rename `EXTERNAL_RENDERERS` to `EXTERNAL_SERVICES` since that is what we call them in the menu -- consistency is good! and thus `getRenderers` into `getExternalServices` and anything else which mentions "render" whenever it might be just giving away the file for torture of a kind we have no clue about